sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_param.json

```diff
@@ -131,31 +131,43 @@
     "spark_runtime_version" : {
         "value": ""
     },
+    "offlineStoreStorageAccountOption": {
+        "value": "none"
+    },
     "offline_store_connection_name" : {
-        "value": ""
+        "value": null
```

Comment:
can you move the "offline_store_connection_name" after the offline_store_subscription_id?

Reply:
done